idx/memory/memory.go

```diff
@@ -639,42 +597,75 @@ func (m *MemoryIdx) Prune(orgId int, oldest time.Time) ([]idx.Archive, error) {
 	return pruned, nil
 }
 
-// filepath.Match doesn't support {} because that's not posix, it's a bashism
-// the easiest way of implementing this extra feature is just expanding single queries
-// that contain these queries into multiple queries, who will be checked separately
-// and whose results will be ORed.
```

Comment:
so your first interpretation was correct. But I only realized now we've stopped using filepatch.Match (via 6dc3776a2b24568d94f3a406677143c82c58fc41) so you're right that it needs to be rephrased at least.